content/telegraf/v1.16/concepts/aggregator_processor_plugins.md

```diff
@@ -43,7 +43,22 @@ Since many users will only care about their aggregates and not every single metr
 gathered, there is also a `drop_original` argument, which tells Telegraf to only
 emit the aggregates and not the original metrics.
 
-**NOTE** Since aggregator plugins only aggregate metrics within their periods,
-historical data is not supported. In other words, if your metric timestamp is more
-than `now() - period` in the past, it will not be aggregated. If this is a feature
-that you need, please comment on this [GitHub issue](https://github.com/influxdata/telegraf/issues/1992).
+{{% note %}}
+#### Aggregator plugins do not support historical data
+Since aggregator plugins only aggregate metrics within their periods,
+historical data is not supported.
+In other words, if your metric timestamp is more
+than `now() - period` in the past, it is not aggregated.
```

Comment:
this doesn't seem to be the case anymore with the `grace` grace period argument, as it's now configurable

Reply:
Looks like that grace period was added in 1.17. Should I leave the note in this one (1.16), but take it out of the 1.17 doc?